scripts/make-multitenant-cluster.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euxo pipefail
```

Comment:
I had the `x` option turned on for development but I've left it on since these scripts will mostly be running in CI or during a release so having the extra logging seems useful.